### PR TITLE
Update System Analytics Controller

### PR DESCRIPTION
**Pull request by sam-warren**
# Fixes or Implements [AB#8898](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8898)

* [X] Enhancement
* [ ] Bug
* [ ] Documentation

## Reviewer Guidelines

A reviewer should examine the PR if the Status above indicates the PR is **Ready**.
Once ready, the reviewer should read the information below as well as supporting information like the Azure ticket to ensure they have a good understanding of the request.

The reviewer should then:

1. Is the PR template complete and accurate?

    Confirm by reading all sections below.

1. Confirm the merge branch

    A normal feature/bug branch should merge into the Dev branch.  Exceptions for hotfix and sprint merges do occur and the target would then be Master.

1. Do the code commits have an appropriate comment and include linkage to the azure ticket by including a reference to "AB#nnnn"? 

1. Review the Code Changes

    1. Do the changes match the description?

        The code commits should reflect the documented description and nothing more.  If additional changes are included the reviewer should call these out and request that the developer document the changes in Azure and in the description or remove the changed code.

    1. useful tips

        * Any obvious errors or omissions?
        * Does the code match the style of other similar work?
        * Is the code easy to understand?
        * Is the code well documented?
        * DB Migrations are appropriate?

If the Answer to any of the questions above is No, then a request for change should be the result of the review.

## Description

Add endpoint to download user ratings

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [X] Not Required

### UI Changes

YES

### Browsers Tested

* [X] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

### CODE REVIEW

File: Apps/AdminWebClient/src/Server/Controllers/CsvExportController.cs [hunk 1]
```diff
@@ -97,6 +97,23 @@ public IActionResult GetNotes(DateTime? startDate = null, DateTime? endDate = nu
...
+        /// Retrieves a list of Ratings inclusively between UTC dates if provided.
+        /// </summary>
+        /// <returns>The invite email.</returns>
+        /// <param name="startDate">The optional start date for the data.</param>
+        /// <param name="endDate">The optional end date for the data.</param>
+        /// <response code="200">Returns the list of beta requests.</response>
+        /// <response code="401">the client must authenticate itself to get the requested response.</response>
+        /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
+        [HttpGet]
+        [Route("GetNotes")]
```
Comment (tiagograf-AOT) [inline Apps/AdminWebClient/src/Server/Controllers/CsvExportController.cs:110]: GetRatings

File: Apps/AdminWebClient/src/Server/Controllers/CsvExportController.cs [hunk 2]
```diff
@@ -97,6 +97,23 @@ public IActionResult GetNotes(DateTime? startDate = null, DateTime? endDate = nu
...
+        [Produces("text/csv")]
+        public IActionResult GetNotes(DateTime? startDate = null, DateTime? endDate = null)
+        {
+            return SendContentResponse("Notes", this.dataExportService.GetNotes(startDate, endDate));
```
Comment (tiagograf-AOT) [inline Apps/AdminWebClient/src/Server/Controllers/CsvExportController.cs:114]: Ratings

File: Apps/AdminWebClient/src/Server/Controllers/CsvExportController.cs [hunk 3]
```diff
@@ -97,6 +97,23 @@ public IActionResult GetNotes(DateTime? startDate = null, DateTime? endDate = nu
...
+        /// <returns>The invite email.</returns>
+        /// <param name="startDate">The optional start date for the data.</param>
+        /// <param name="endDate">The optional end date for the data.</param>
+        /// <response code="200">Returns the list of beta requests.</response>
+        /// <response code="401">the client must authenticate itself to get the requested response.</response>
+        /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
+        [HttpGet]
+        [Route("GetNotes")]
+        [Produces("text/csv")]
+        public IActionResult GetNotes(DateTime? startDate = null, DateTime? endDate = null)
```
Comment (tiagograf-AOT) [inline Apps/AdminWebClient/src/Server/Controllers/CsvExportController.cs:112]: GetRatings

File: Apps/AdminWebClient/src/Server/Controllers/CsvExportController.cs [hunk 4]
```diff
@@ -97,6 +97,23 @@ public IActionResult GetNotes(DateTime? startDate = null, DateTime? endDate = nu
             return SendContentResponse("Notes", this.dataExportService.GetNotes(startDate, endDate));
         }
 
+        /// <summary>
+        /// Retrieves a list of Ratings inclusively between UTC dates if provided.
+        /// </summary>
+        /// <returns>The invite email.</returns>
```
Comment (leobcgov) [inline Apps/AdminWebClient/src/Server/Controllers/CsvExportController.cs:103]: update the returns param

**Comment by leobcgov** [comment]
@sam-warren as the build of your PR is failing, mbe wait until my PR is approved (https://github.com/bcgov/healthgateway/pull/741) then pull dev to your PR?

**Comment by sam-warren** [comment]
> @sam-warren as the build of your PR is failing, mbe wait until my PR is approved (#741) then pull dev to your PR?

The build fails because it is looking for tests and not finding them. Once Mandad's PRs are merged we won't have build issues

**Pull request merged**